package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-keyboard-aware-scroll-view",
-  "version": "0.8.8-wp",
+  "version": "0.8.8-wp-1",
```

Comment:
This version doesn't have a Git tag so we should create it once this gets merged.